backend/Builtins.icl

```diff
@@ -463,6 +464,16 @@ bs_module = (["module", "definition", "implementation", "system", "definition mo
 		]
 	})
 
+bs_pattern_named = (["=:"],
+	{ syntax_title         = "newtype definition (experimental)"
+	, syntax_code          = [":: ... =: T c"]
```

Comment:
`T` is not required, so more general would be `:: ... =: ... ...` (and the examples can explain how to use it)